pkg/chain/ethereum/ethereum.go

```diff
@@ -516,24 +514,20 @@ func (ec *EthereumChain) GetOpenedTimestamp(keepAddress common.Address) (time.Ti
 // for the given keep which occurred after the provided start block.
 // Returned events are sorted by the block number in the ascending order.
 func (ec *EthereumChain) PastSignatureSubmittedEvents(
-	keepAddress string,
+	keepAddress common.Address,
 	startBlock uint64,
 ) ([]*eth.SignatureSubmittedEvent, error) {
-	if !common.IsHexAddress(keepAddress) {
-		return nil, fmt.Errorf("invalid keep address: [%v]", keepAddress)
-	}
```

Comment:
We should accept `string` and this validation should stay.

Reply:
Done.